### DISCUSSION TITLE
連「天安門」都不敢翻譯的中國 AI 國家隊，「科大訊飛」稱十月將超越 ChatGPT

### DISCUSSION THREAD

**Discussion by zephyrxin01** [1101-GS4719-C/Python-Community-GS4719-C#1388] [General]
- [連「天安門」都不敢翻譯的中國 AI 國家隊，「科大訊飛」稱十月將超越 ChatGPT](https://buzzorange.com/techorange/2023/05/08/iflyteck-ai/)
- 中國的「訊飛星火認知大模型」宣稱他們已經在文本生成、知識問答、數學能力三大能力上超過 ChatGPT。董事長劉慶峰表示他們的模型在今年10月底將以「中文超越，英文相當！」的氣魄趕超ChatGPT。
- 討論
    - 各位怎麼看待科大訊飛宣稱他們將超越 ChatGPT?
    - 大家覺得使用ChatGPT時，用英文生成和中文生成回答有很大的落差嗎?
    - 你們有使用升級付費的ChatGPT嗎?
- 我的看法
    - 我覺得名字取的不夠旺，記憶點似乎不太夠。
    - 我自己在人機合作的時候其實沒有特別感覺到中英文有什麼太大的差別。因為不論是中文或是英文的回答，我都還是需要自己再潤飾過才不會覺得整個語句不太連貫。
    - 沒有QQ，首先沒有預算，所以目前還是先靠自己。但我蠻好奇其他同學如果有使用，會應用在什麼方向上。

**Comment by TW-Maverick** [comment]
- 如果是說數學能力的話，在ChatGPT中有一個新的套件「Wolfram」，它改進之前ChatGPT在處理運算或數學問題時遇到的Bug，但我不能保證他會比我們偉大的中國AI 國家隊所做的˙模型來的更好。👌

- 如果要問ChatGPT旅遊或吃飯建議的話，我會建議使用英文，因為回答通常比較好。

- 我沒有升級，沒錢QQ

以上是小弟的鄙見 ，請各位多多指正。


<details><summary>連結</summary>
<p>

🔗參考連結
[ChatGPT plugins](https://openai.com/blog/chatgpt-plugins)

[Wolfram Plugin for ChatGPT](https://www.wolfram.com/wolfram-plugin-chatgpt/)

</p>
</details>
